src/components/PostList/PostList.tsx

```diff
@@ -1,5 +1,13 @@
 import React from 'react';
+import { Post } from '../../types/types';
+import { PostInfo } from '../PostInfo';
 
-export const PostList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
```

Comment:
nit: Consider using an interface instead of a type for component Props. It's more idiomatic for React components 🧐.

Reply:
Sorry, but I can't find any idea how to replace the type on the interface. Could you please help me with this?

Reply:
try using 
```
interface Props {
  ...
}
```

instead of 
```
type Props = {

}
```